[2025-03-21T16:41:13.472+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_map.load_phase_task manual__2025-03-21T16:40:09.876807+00:00 [queued]>
[2025-03-21T16:41:13.495+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_map.load_phase_task manual__2025-03-21T16:40:09.876807+00:00 [queued]>
[2025-03-21T16:41:13.496+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-21T16:41:13.548+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_phase_task> on 2025-03-21 16:40:09.876807+00:00
[2025-03-21T16:41:13.556+0000] {standard_task_runner.py:60} INFO - Started process 5362 to run task
[2025-03-21T16:41:13.563+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'Google_map', 'load_phase_task', 'manual__2025-03-21T16:40:09.876807+00:00', '--job-id', '277', '--raw', '--subdir', 'DAGS_FOLDER/google_map_dag_etl.py', '--cfg-path', '/tmp/tmpol2adc56']
[2025-03-21T16:41:13.567+0000] {standard_task_runner.py:88} INFO - Job 277: Subtask load_phase_task
[2025-03-21T16:41:13.712+0000] {task_command.py:423} INFO - Running <TaskInstance: Google_map.load_phase_task manual__2025-03-21T16:40:09.876807+00:00 [running]> on host DESKTOP-AVAHALM.
[2025-03-21T16:41:13.927+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='master_m2si' AIRFLOW_CTX_DAG_ID='Google_map' AIRFLOW_CTX_TASK_ID='load_phase_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T16:40:09.876807+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-21T16:40:09.876807+00:00'
[2025-03-21T16:41:13.930+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-21T16:41:13.931+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd ~/data_warehouse_project && dbt run --profiles-dir ~/.dbt --profile decisionnelle_profile --models dim_bank dim_branch dim_location dim_sentiment fact_reviews']
[2025-03-21T16:41:13.946+0000] {subprocess.py:86} INFO - Output:
[2025-03-21T16:41:19.078+0000] {subprocess.py:93} INFO - [0m16:41:19  Running with dbt=1.9.0-b2
[2025-03-21T16:41:19.896+0000] {subprocess.py:93} INFO - [0m16:41:19  Registered adapter: postgres=1.8.2
[2025-03-21T16:41:20.327+0000] {subprocess.py:93} INFO - [0m16:41:20  Unable to do partial parsing because config vars, config profile, or config target have changed
[2025-03-21T16:41:20.329+0000] {subprocess.py:93} INFO - [0m16:41:20  Unable to do partial parsing because profile has changed
[2025-03-21T16:41:26.749+0000] {subprocess.py:93} INFO - [0m16:41:26  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-21T16:41:26.750+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-03-21T16:41:26.750+0000] {subprocess.py:93} INFO - - models.data_warehouse_project.example
[2025-03-21T16:41:27.349+0000] {subprocess.py:93} INFO - [0m16:41:27  Found 6 models, 1 source, 425 macros
[2025-03-21T16:41:27.354+0000] {subprocess.py:93} INFO - [0m16:41:27
[2025-03-21T16:41:27.356+0000] {subprocess.py:93} INFO - [0m16:41:27  Concurrency: 4 threads (target='dev_decisionnelle')
[2025-03-21T16:41:27.358+0000] {subprocess.py:93} INFO - [0m16:41:27
[2025-03-21T16:41:27.836+0000] {subprocess.py:93} INFO - [0m16:41:27  1 of 5 START sql table model Decisionnelle.dim_bank ............................ [RUN]
[2025-03-21T16:41:27.848+0000] {subprocess.py:93} INFO - [0m16:41:27  2 of 5 START sql table model Decisionnelle.dim_branch .......................... [RUN]
[2025-03-21T16:41:27.855+0000] {subprocess.py:93} INFO - [0m16:41:27  3 of 5 START sql table model Decisionnelle.dim_location ........................ [RUN]
[2025-03-21T16:41:27.861+0000] {subprocess.py:93} INFO - [0m16:41:27  4 of 5 START sql table model Decisionnelle.dim_sentiment ....................... [RUN]
[2025-03-21T16:41:29.471+0000] {subprocess.py:93} INFO - [0m16:41:29  3 of 5 OK created sql table model Decisionnelle.dim_location ................... [[32mSELECT 2[0m in 1.59s]
[2025-03-21T16:41:29.481+0000] {subprocess.py:93} INFO - [0m16:41:29  4 of 5 OK created sql table model Decisionnelle.dim_sentiment .................. [[32mSELECT 2[0m in 1.61s]
[2025-03-21T16:41:29.492+0000] {subprocess.py:93} INFO - [0m16:41:29  1 of 5 OK created sql table model Decisionnelle.dim_bank ....................... [[32mSELECT 2[0m in 1.63s]
[2025-03-21T16:41:29.501+0000] {subprocess.py:93} INFO - [0m16:41:29  2 of 5 OK created sql table model Decisionnelle.dim_branch ..................... [[32mSELECT 2[0m in 1.64s]
[2025-03-21T16:41:29.508+0000] {subprocess.py:93} INFO - [0m16:41:29  5 of 5 START sql table model Decisionnelle.fact_reviews ........................ [RUN]
[2025-03-21T16:41:29.784+0000] {subprocess.py:93} INFO - [0m16:41:29  5 of 5 OK created sql table model Decisionnelle.fact_reviews ................... [[32mSELECT 9[0m in 0.27s]
[2025-03-21T16:41:29.836+0000] {subprocess.py:93} INFO - [0m16:41:29
[2025-03-21T16:41:29.837+0000] {subprocess.py:93} INFO - [0m16:41:29  Finished running 5 table models in 0 hours 0 minutes and 2.48 seconds (2.48s).
[2025-03-21T16:41:30.075+0000] {subprocess.py:93} INFO - [0m16:41:30
[2025-03-21T16:41:30.076+0000] {subprocess.py:93} INFO - [0m16:41:30  [32mCompleted successfully[0m
[2025-03-21T16:41:30.077+0000] {subprocess.py:93} INFO - [0m16:41:30
[2025-03-21T16:41:30.078+0000] {subprocess.py:93} INFO - [0m16:41:30  Done. PASS=5 WARN=0 ERROR=0 SKIP=0 TOTAL=5
[2025-03-21T16:41:31.970+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-21T16:41:32.144+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Google_map, task_id=load_phase_task, execution_date=20250321T164009, start_date=20250321T164113, end_date=20250321T164132
[2025-03-21T16:41:32.280+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-21T16:41:32.333+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
